superset/views/base.py

```diff
@@ -96,6 +98,20 @@ def json_error_response(
     )
 
 
+def json_errors_response(
+    errors: List[SupersetError], status: int = 500, payload: Optional[dict] = None,
+) -> Response:
+    if not payload:
+        payload = {}
+
+    payload["errors"] = [dataclasses.asdict(error) for error in errors]
+    return Response(
```

Comment:
Could `json_errors_response` or similar work here? Ideally we use the same mechanism for constructing JSON responses.

Reply:
huh? this is inside `json_errors_response`

Reply:
🤦